src/Symfony/Bridge/Doctrine/Tests/Form/EventListener/MergeDoctrineCollectionListenerTestCase.php

```diff
@@ -20,7 +20,7 @@
 use Symfony\Component\Form\FormEvents;
 use Symfony\Component\Form\FormFactoryInterface;
 
-class MergeDoctrineCollectionListenerTest extends TestCase
+class MergeDoctrineCollectionListenerTestCase extends TestCase
```

Comment:
Is this change necessary as this class is not abstract (same for other classes below)?